src/core/server.cc

```diff
@@ -1030,6 +1030,12 @@ InferenceServer::Wait()
                         << "': " << status.error_message();
               goto next;
             }
+            status = status_manager_->InitForModel(name);
```

Comment:
Note the comment above about not wanting to re-init the status since that will delete all the stats collected for the model. What we need is a net ServerStatusManager method that just updates the model configuration in the status but doesn't clear the existing status.

Reply:
What about **ms[model_name].Clear()** in InitForModel?

Reply:
Oh, you are right. I missed that.